aicspylibczi/_version.py

```diff
@@ -0,0 +1,9 @@
+# This file is part of aicspylibczi.
+#
+# You should have received a copy of the GNU General Public License
+# along with aicspylibczi.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Version information."""
+
+# The following line *must* be the last in the module, exactly as formatted:
+__version__ = "2.2.0"
```

Comment:
Have you already bumpversioned? If so, I would tag the commit after merging to master.